cms/templates/settings.html

```diff
@@ -292,6 +292,28 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <span class="tip">${_("Information for prospective students")}</span>
               </header>
               <ol class="list-input">
+
+                % if about_page_editable:
```

Comment:
Let's wrap all of these fields in a single `% if about_page_editable:`. We can put the Short Description field at the end.